src/intersection-observer.ts

```diff
@@ -174,26 +180,32 @@ export function generateEntry(
   clientRect: DOMRectReadOnly,
   el: HTMLElement,
   rootMargin: DOMMargin
-): IntersectionObserverEntry {
+): InternalIntersectionObserverEntry {
   if (el.style.display === 'none') {
     return {
+      unixTime: frame.dateNow,
```

Comment:
Same above. Can we keep only one of `time` and `unixTime`

Reply:
removed